mainWindow.py

```diff
@@ -22,7 +23,7 @@ class mainWindow(QMainWindow, QMessageBox):
     def __init__(self, fileA=0, fileB=0):
         super().__init__()
         self.setWindowTitle("PyMerge")
-        self.setGeometry(1000, 1000, 2000, 1000)
+        self.setGeometry(100, 100, 1000, 800)
```

Comment:
Could use some comments giving reasons for choosing a specific set of dimensions; just so they don't seem randomly picked.